doc/man/salt.7

```diff
@@ -428614,6 +428608,12 @@ Mark this host as trusted, even though it does not have valid or any
 HTTPS.
 .UNINDENT
 .sp
+.B bin_env
```

Comment:
The man pages automatically get generated. can you remove the man page changes please

Reply:
Sure @Ch3LL , sorry i added it after pre-commit generating it, doing it right now.